### PR TITLE
Fix for issue Refactoring of bequeath-your-data-and-die as per the flow followed in other TinyApplicationControllers #31

### PR DESCRIPTION
**Pull request by vanithavalluripalli9**
Fixes issue Refactoring of bequeath-your-data-and-die as per the flow followed in other TinyApplicationControllers #31

### CODE REVIEW

File: server/service/individualServices/PrepareForwardingAutomation.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const prepareALTForwardingAutomation = require('onf-core-model-ap-bs/basicServices/services/PrepareALTForwardingAutomation');
+
+exports.bequeathYourDataAndDie = function (logicalTerminationPointconfigurationStatus) {
+    return new Promise(async function (resolve, reject) {
```
Comment (MartinSunal) [inline server/service/individualServices/PrepareForwardingAutomation.js:4]: please, use indentation 2 spaces as in other files

File: server/service/individualServices/SoftwareUpgrade.js [hunk 2]
```diff
@@ -0,0 +1,664 @@
+/**
+ * @file This module provides functionality to migrate the data from the current version to the next version. 
+ * @module SoftwareUpgrade
+ **/
+
+ const operationClientInterface = require('onf-core-model-ap/applicationPattern/onfModel/models/layerProtocols/OperationClientInterface');
```
Comment (MartinSunal) [inline server/service/individualServices/SoftwareUpgrade.js:6]: every line in this file has a space at the beginning. Please, remove them and keep indentation 2 spaces.

**Review by MartinSunal — CHANGES_REQUESTED** [review]

File: server/service/individualServices/PrepareForwardingAutomation.js (hunk 1 above)
Comment (vanithavalluripalli9) [reply server/service/individualServices/PrepareForwardingAutomation.js:4]: fixed the issue

File: server/service/individualServices/SoftwareUpgrade.js (hunk 2 above)
Comment (vanithavalluripalli9) [reply server/service/individualServices/SoftwareUpgrade.js:6]: fixed the issue

File: server/service/individualServices/SoftwareUpgrade.js [hunk 3]
```diff
@@ -0,0 +1,664 @@
...
+ * of the application and bring the new release official
+ * @param {boolean} isdataTransferRequired represents true if data transfer is required
+ * @param {String} user User identifier from the system starting the service call
+ * @param {String} xCorrelator UUID for the service execution flow that allows to correlate requests and responses
+ * @param {String} traceIndicator Sequence of request numbers along the flow
+ * @param {String} customerJourney Holds information supporting customer’s journey to which the execution applies
+   @returns {Promise} Promise is resolved if the operation succeeded else the Promise is rejected 
+* **/
+exports.upgradeSoftwareVersion = async function (isdataTransferRequired, user, xCorrelator, traceIndicator, customerJourney) {
+    return new Promise(async function (resolve, reject) {
```
Comment (MartinSunal) [inline server/service/individualServices/SoftwareUpgrade.js:30]: as mentioned in my previous comment - please, use indentation 2 spaces everywhere

**Review by MartinSunal — APPROVED** [review]

**Pull request merged**